src/components/GalleryEditor/CollectionEditor/StagingArea.tsx

```diff
@@ -205,44 +205,46 @@ function StagingArea({ tokensRef }: Props) {
   const hasNameOrCollectorsNote = name || collectorsNote;
   return (
     <StyledStagingArea gap={20} align="center">
-      <CollectionNameAndDescriptionContainer justify="start" align="center">
```

Comment:
small bug fix, basically we don't want to show a collection title / description form if you don't have a collection selected